org.eclipse.capra.ui.plantuml/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+/target
```

Comment:
Yes, I added it because Eclipse has previously created these automatically for me. It is also consistent with the other plugins in the project, and with what amalthea-src did, and I believe other Eclipse projects.

However it didn't create it for me now when I tried to repeat it. Perhaps it has to do with how you import the projects. 

If it isn't required I suggest we create a separate pull-request to remove all plugin .gitignores to keep it consistent.